src/mysql_protocol.erl

```diff
@@ -440,7 +441,8 @@ fetch_response(SockModule, Socket, Timeout, Proto, Acc) ->
                     parse_ok_packet(Packet);
                 ?error_pattern ->
                     parse_error_packet(Packet);
-                <<251:8, FileName/binary>> -> 
+                ?local_local_data_file ->
```

Comment:
Good, but the name contains the word "local" twice. :-)

Reply:
Actually, I think it's enough (and more clear) to use `<<?LOAD_LOCAL_FILE_REQ, FileName/binary>> ->` directly here. It's a very simple structure here. (For the others cases, e.g. `?error_pattern`, it was a more complex structure of so I preferred a macro for the pattern and a separate function to parse the packet.)

The name `LOAD_LOCAL_FILE_REQ` is good.